Makefile

```diff
@@ -56,6 +56,11 @@ build-env: # Make a new conda environment
 install: # Install setuptools, install this package in editable mode
 	pip install --upgrade pip setuptools
 	pip install -e .[DEV]
+	@echo "Checking if the Layered Config Tree repository has the branch $(GIT_BRANCH)..."
+	@$(eval BRANCH_EXISTS=$(shell curl -s https://api.github.com/repos/ihmeuw/layered_config_tree/branches | grep -q '"name": "$(GIT_BRANCH)"' && echo "yes" || echo "no"))
+	@if [ "$(BRANCH_EXISTS)" = "yes" ]; then \
```

Comment:
Is it possible to use the bash script I created earlier this week to check for and install branches?

Reply:
I was just thinking about this. I can play around with it and see